examples/lambda/README.md

```diff
@@ -0,0 +1,9 @@
+# Lambda Example
+
+This examples shows how to deploy an AWS Lambda function. 
```

Comment:
I would like to enhance this message with: `This examples shows how to deploy an AWS Lambda function using Terraform only.`

Just to expose my point of view: another way to deploy a given lambda is by building the ZIP in your CI, put it on S3, and use a s3 object data source to get the file.
With this logic, you can version your lambdas and benefit from a real build - deploy phase.

Thoughts? :)

Reply:
I think that's an improvement. I'll add it.